app/utils/callback/lava.py

```diff
@@ -387,6 +387,14 @@ def _add_log_fragments(self):
 
     @property
     def start_log_line(self):
+        """ Get line number at which log starts
```

Comment:
I believe this code is being added as part of this PR, so this commit could be squashed with the commit that adds it.

Reply:
squashed